runtime/kusama/src/lib.rs

```diff
@@ -317,7 +317,7 @@ parameter_types! {
 	// 27 eras in which slashes can be cancelled (slightly less than 7 days).
 	pub const SlashDeferDuration: pallet_staking::EraIndex = 27;
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
-	pub const MaxNominatorRewardedPerValidator: u32 = 64;
+	pub const MaxNominatorRewardedPerValidator: u32 = 128;
```

Comment:
```suggestion
	pub const MaxNominatorRewardedPerValidator: u32 = 256;
```

I don't see a reason why not to.